tasks/2017/interactive-bots.md

```diff
@@ -35,92 +35,56 @@ messages and can be integrated with various APIs.
-
-  If you're running your bot in the droplet, alongside your Zulip instance,
-  the `<dev-url>` will be `localhost:9991`.
-
-* Create a `followup` stream on your Zulip server.
-
-* Use `Manage Streams` to subscribe your bot (if you created one) to the
-following streams: `devel`, `social`, and `followup`.
-
-* Run the followup bot:
-
-  ```
-  cd ~/zulip/contrib_bots
-  python run.py lib/followup.py --config-file ~/.zuliprc-local
-  ```
-
-    Make sure to point to your `.zuliprc-local` file - if you have created
-    it in a different folder than `~`, you have to point to the right folder,
-    e.g. `../../.zuliprc-local`.
-
-    **Note:** if you're using Vagrant, make sure you run the bot **outside**
-    of the Vagrant container.
-
-
-* Test manually that the followup bot is working on the local Zulip instance,
+* Test manually that the helloworld bot is working on the local Zulip instance,
 by:
 
- - sending a few messages starting with `@followup` and `@follow-up` from
- different streams, such as `devel` and `social`
+ - sending a few messages starting with `@<your-name>_helloworld_bot` from
+ the stream you subscribed the bot to.
 
- - checking that the messages showed up in the `followup` stream, prepended
- by the sender email
+ - checking that the bot is replying with `beep boop` message each time.
 
 * Take screenshots showing that the bot is working, make sure to have
 screenshots of:
 
  - your terminal window with the bot running, including the command you used
  to run the bot and the output with the bot description
- - messages sent by you to the bot in the `devel` and `social` streams
- - messages sent by the bot in the `followup` stream
+ - messages sent by you to the bot in the subscribed stream
+ - messages sent by the bot in the same stream
  - any other screenshots you find relevant
 
-  Add the screenshots to `interactive-bots/followup/<username>/`. Make sure
+  Add the screenshots to `interactive-bots/helloworld/<username>/`. Make sure
   your filenames do not have white-spaces. Instead, use dashes (`-`).
 
 * Note down any places you got stuck, problems or errors you ran into while
 doing this setup process. Add your notes as a `notes.md` file to
-`interactive-bots/followup/<username>/`.
+`interactive-bots/helloworld/<username>/`.
 
 * Create a commit with the screenshots and notes, with commit message
-`interactive bots: Run the followup bot for <username>.`.
+`interactive bots: Run the helloworld bot for <username>.`.
 
-* Create a pull request in the `zulip/zulip-gci` repository, with title
-`interactive bots: Run the followup bot for <username>`. Link to your GCI
+* Create a pull request in the `zulip/zulip-gci-submissions-2017` repository, with title
+`interactive bots: Run the helloworld bot for <username>`. Link to your GCI
 task in comment section.
 
-*Completion criteria:* Mentors will check that the followup bot was properly
+*Completion criteria:* Mentors will check that the helloworld bot was properly
```

Comment:
We should use consistent quoting for referring to the `helloworld` bot (sometimes it's "", sometimes '' and here without quotes). I'd just go without quotes everywhere.

Reply:
Ah, interesting catch!